Configuration.props

```diff
@@ -37,10 +51,16 @@
     <AllSupportedTargetAndroidAbis>$(AllSupported32BitTargetAndroidAbis);$(AllSupported64BitTargetAndroidAbis)</AllSupportedTargetAndroidAbis>
   </PropertyGroup>
   <PropertyGroup>
+    <_MingwPrefixTail Condition=" '$(HostOS)' == 'Darwin' ">.static</_MingwPrefixTail>
+    <_MingwCommandPrefix32>i686-w64-mingw32$(_MingwPrefixTail)</_MingwCommandPrefix32>
```

Comment:
`Configuration.props` shouldn't "export" `_`-prefixed values. Convention is that a `_` prefix means "private" -- though nothing enforces that! -- so if something is intended to be used from other files, it shouldn't have a `_` prefix.